README.md

```diff
@@ -1 +1,198 @@
+[![codecov](https://codecov.io/gh/wayfair-incubator/extra-model/branch/main/graph/badge.svg?token=HXSGN5IUzu)](https://codecov.io/gh/wayfair-incubator/extra-model)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=ef8336)](https://pycqa.github.io/isort/)
+
+TODO: update badges
+
 # extra-model
+
+Code to run the Extra [algorithm](https://aclanthology.info/papers/D18-1384/d18-1384) for unsupervised topic extraction.
+
+## Quick start
+
+**IMPORTANT**: 
+1. When running Extra inside docker-container, make sure that Docker process has enough resources. 
+For example, on Mac/Windows it should have at least 8 Gb of RAM available to it.
+1. GitHub repo does **not** come with Glove Embeddings. TODO: update after script is in place.
+
+### Using docker-compose
+
+First, build the image:
+
+```bash
+docker-compose build
+```
+
+Then running `extra-model` is as simple as:
+
+```bash
+docker-compose run dev python /io/extra_model/_cli.py /io/tests/resources/100_comments.csv
```

Comment:
this will be updated when CLI exposes its entrypoint on python package install, i'm working on this now 👍